### DISCUSSION TITLE
Channel Map

### DISCUSSION THREAD

**Discussion by DevastatorCM** [Khopa/dcs_liberation#2216] [Q&A] [answered]
Is there a way to play as Axis in the channel map campaign and use the mainland airfields as bases? When I switch the factions, Axis bases are located in Britain...

**Comment by Starfire13** [comment]
You can use any faction for any campaign. If you are finding the bases flipped, it's probably because you have ticked the invert campaign checkbox.

**Comment by DevastatorCM** [comment]
I did not pick the invert map. If I pick that it gives error anyway.
When I do not pick it look what I see..
http://prntscr.com/ZyIzAfa4-LwG
so I can buy BF109 in Hawkinge airbase...

**Reply by Starfire13** [reply, marked as answer]
Oh I get what you mean now. Sorry, reading comprehension fail on my part. You're saying you want the airfields inverted. In that case, you'll have to set it up yourself by editing the campaign template (instructions are on the wiki; see the custom campaign section). Not all campaign designers implement inversion for their campaigns as it is an optional feature (and campaign balance is typically out of whack after inversion anyway).
